examples/demo.rs

```diff
@@ -37,7 +38,7 @@ fn console_output(results: &Option<Vec<GpuTimerScopeResult>>, enabled_features:
 async fn run(event_loop: EventLoop<()>, window: Window) {
     let size = window.inner_size();
     let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
-        backends: wgpu::Backends::all(),
+        backends: wgpu::Backends::DX12,
```

Comment:
It looks like this was changed for testing and needs to be switched back to `all` or at least to include vulkan.